tests/runners/dataset_runner.py

```diff
@@ -70,3 +71,40 @@ def upload_spreadsheet_and_create_submission(self, dataset_fixture, project_uuid
                                                        project_uuid=project_uuid)
         Progress.report(f"submission is in {self.ingest_api.ingest_api_url}/submissionEnvelopes/{self.submission_id}\n")
         self.submission_envelope = self.ingest_api.envelope(self.submission_id)
+
+    def __submit_archive_submission(self, dataset_fixture):
+        self.valid_run(dataset_fixture)
+        self.submission_manager.submit_envelope(["Archive"])
+        self.submission_manager.wait_for_envelope_to_be_archiving()
+        projects = self.submission_envelope.retrieve_projects()
+        if len(projects) > 0:
+            project = projects[0]
+            project_url = project.get_url()
+            now = datetime.now()
+            r = self.ingest_client_api.patch(project_url, {'releaseDate': now.strftime('%Y-%m-%dT%H:%M:%SZ')})
```

Comment:
I think I saw this time format in several locations.

Should it be a constant?

Reply:
Created a constant, although it was only used here, so only 1 appearance.